options/locale/locale_en-US.ini

```diff
@@ -3407,6 +3407,8 @@ runners.status.offline = Offline
 runners.version = Version
 runners.reset_registration_token_success = Runner registration token reset successfully
 
+artifacts.title = Artifacts
```

Comment:
Still not top-level, I mean top level as in between lines 1 and 116 in the file, e.g. no prefix.

Reply:
Thanks, looking good now.